backend/studyagent/taskmaster/google.py

```
@@ -0,0 +1,198 @@
+        private = {"studyagent_key": key, "run_id": run_id}
+        if key.startswith("deadline:"):
+            # The deadline body is built from a Task, whose provenance is added
+            # by sync() before calling this helper.
+            private.update(body.pop("_private", {}))
+        body["extendedProperties"] = {"private": private}
+        payload = {"key": key, "state": "writing", "desired_hash": digest, "run_id": run_id, "attempted_at": datetime.now(UTC)}
+        if audit:
+            payload.update(audit)
+        ref.set(payload, merge=True)
+        if binding.get("google_event_id"):
+            result = service.events().patch(calendarId=calendar_id, eventId=binding["google_event_id"], body=body).execute(); action = "updated"
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)